packages/core/src/layout/ErrorPage/ErrorPage.tsx

```diff
@@ -48,6 +53,8 @@ export const ErrorPage = ({ status, statusMessage }: IErrorPageProps) => {
       <Grid item xs={12} sm={4}>
         <Typography variant="body1" className={classes.subtitle}>
           ERROR {status}: {statusMessage}
+          <br />
```

Comment:
Typically you'd use several `<Typography paragraph>` instead, which makes sure that spacing etc is correct